### PR TITLE
Remove inaccurate comment about adversarial loss in wgan-gp

### PR DESCRIPTION
**Pull request by konstantinjdobler**
Hello,

thank you for your implementations. In WGAN-GP, there is an inaccurate comment about the loss used. It is not a binary cross-entropy loss, but rather the negated mean of the discriminator logits.

### CODE REVIEW

**Review by nocotan — APPROVED** [review]
LGTM
